games/static/games/proto.js

```diff
@@ -51,7 +75,7 @@ class Square {
     }
 
     isTop() {
-        return this.index <= this.lineWidth
+        return this.index < this.lineWidth
     }
 
     setCharacter(character) {
```

Comment:
i wonder: `this.character.square = this.index` 
is square on character is actually an int?
if so, i wouldnt expect that
maybe it should be named squareIndex?

Reply:
? i would need clarification on this one

Reply:
this line was redundant and i erased it